src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -43,6 +44,10 @@ public int countMatchingNumbersWithWinningTicket(LottoTicket winningLottoTicket)
                 .count();
     }
 
+    public boolean matchesBonusNumber(BonusNumber bonusNumber) {
+        return numbers.stream().anyMatch(bonusNumber::matchesBonusNumber);
```

Comment:
객체지향 생활 체조 원칙의 `규칙 4: 한 줄에 점을 하나만 찍는다.`를 적용해보면 어떨까요?
```suggestion
        return numbers.stream()
            .anyMatch(bonusNumber::matchesBonusNumber);
```